File: src/main/java/me/rasztabiga/fridgy/products/repository/AuthorityRepository.java
package me.rasztabiga.fridgy.products.repository;

import me.rasztabiga.fridgy.products.domain.Authority;
import org.springframework.data.jpa.repository.JpaRepository;

/**
 * Spring Data JPA repository for the {@link Authority} entity.
 */
public interface AuthorityRepository extends JpaRepository<Authority, String> {}

File: src/main/kotlin/me/rasztabiga/fridgy/products/repository/AuthorityRepository.kt
package me.rasztabiga.fridgy.products.repository

import me.rasztabiga.fridgy.products.domain.Authority
import org.springframework.data.jpa.repository.JpaRepository

/**
 * Spring Data JPA repository for the [Authority] entity.
 */

interface AuthorityRepository : JpaRepository<Authority, String>
